libs/vkjson/vkjson.cc

```diff
@@ -34,11 +34,16 @@
 
 #include <cJSON.h>
 #include <vulkan/vk_sdk_platform.h>
+#include <stdlib.h>
 
 namespace {
 
 inline bool IsIntegral(double value) {
+#if defined(ANDROID)
+  return trunc(value) == value;
```

Comment:
Android toolchain won't support explicit namespace qualification?
Is there an implicit using decl, or is there no std namespace at all, etc?